hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+Fix :func:`~hypothesis.strategies.from_type`
+on :class:`collections.abc.Callable` returning :const:`None`.
```

Comment:
```suggestion
on :class:`collections.abc.Callable` returning ``None``.
```
Sphinx didn't resolve this; I think we can assume people know about None so the fix can be simple.